cloudformation/iam_federation.js

```diff
@@ -92,15 +105,16 @@ function removeStatement(input, callback) {
   console.log(assumeDoc.Statement);
   console.log(input.roleArn);
   var found = -1;
-  for (var i = 0; i < assumeDoc.Statement.length; i++) {
```

Comment:
What I'm addressing is the old ones that were added before this change like awsconfig/cloudtrail projects.